styles/global.module.css

```diff
@@ -0,0 +1,12 @@
+.pepoButton {
+  color: black;
+  background-color: #ffcc00;
+  margin: 1em;
+  box-shadow: 0px 3px 1px -2px rgba(0, 0, 0, 0.2),
```

Comment:
We should add these `box-shadow` values to the theme file as well and reference that!